source/plugins/go/input/containerinventory/containerinventory.go

```diff
@@ -165,7 +165,9 @@ func (p containerInventoryPlugin) enumerate() []map[string]interface{} {
 					container[k] = v
 				}
 				container["State"] = "Deleted"
-				lib.DeleteCGroupCacheEntryForDeletedContainer(container["InstanceID"].(string))
+				if strings.EqualFold(osType, "windows") {
```

Comment:
why only for windows?

Reply:
Ohh good catch. It had to be only for Linux as windows doesnt have cgroup cache